bitnami/elasticsearch/values.yaml

```diff
@@ -427,23 +462,26 @@ ingest:
     requests:
       cpu: 25m
       memory: 256Mi
+
   ## Elasticsearch ingest container's liveness and readiness probes
   ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
   ##
   livenessProbe:
-    enabled: false
-  #  initialDelaySeconds: 90
-  #  periodSeconds: 10
-  #  timeoutSeconds: 5
-  #  successThreshold: 1
-  #  failureThreshold: 5
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 10
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+
   readinessProbe:
-    enabled: false
-  #  initialDelaySeconds: 90
-  #  periodSeconds: 10
-  #  timeoutSeconds: 5
-  #  successThreshold: 1
-  #  failureThreshold: 5
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 10
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
```

Comment:
Why did you uncomment this block (and others probes)?

Reply:
the probes use these as the defaults https://github.com/bitnami/charts/blob/master/bitnami/elasticsearch/templates/master-statefulset.yaml#L84-L102